src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -53,11 +58,16 @@ internal static void ProcessToolReferences(ProjectNames projectNames, IVsProject
                     dgSpec.AddProject(ts);
                 });
         }
+        #endregion ToolReferenceAPIs
 
-        internal static RuntimeGraph GetRuntimeGraph(IVsProjectRestoreInfo projectRestoreInfo)
+        #region IVSTargetFrameworksAPIs
+        /**********************************************************************
+         * IVSTargetFrameworks based APIs (1st iteration of the nominate API.) * 
```

Comment:
`IVsTargetFrameworks` (casing)

Is `IVsTargetFrameworks2` part of the 2nd iteration? I don't see a comment or region for it like you added for the first iteration.

Reply:
The region covers the family of IVSTargetFrameworks.

Reply:
Then should `(1st iteration of the nominate API.)` be removed?

Reply:
Oh yeah, will do.